vars/uiVeri5ExecuteTests.groovy

```diff
@@ -0,0 +1,152 @@
+import com.sap.piper.ConfigurationHelper
+import com.sap.piper.GenerateDocumentation
+import com.sap.piper.GitUtils
+import com.sap.piper.Utils
+
+import groovy.text.SimpleTemplateEngine
+import groovy.transform.Field
+
+import static com.sap.piper.Prerequisites.checkScript
+
+@Field String STEP_NAME = getClass().getName()
+
+@Field Set GENERAL_CONFIG_KEYS = [
+    /**
+     * In case a `testRepository` is provided and it is protected, access credentials (as Jenkins credentials) can be provided with `gitSshKeyCredentialsId`. **Note: In case of using a protected repository, `testRepository` should include the ssh link to the repository.**
+     * @possibleValues Jenkins credentialId
+     */
+    'gitSshKeyCredentialsId'
+]
+@Field Set STEP_CONFIG_KEYS = GENERAL_CONFIG_KEYS.plus([
+    /**
+     * A map of environment variables to set in the container, e.g. [http_proxy:'proxy:8080'].
+     */
+    'dockerEnvVars',
+    /**
+     * The name of the docker image that should be used. If empty, Docker is not used and the command is executed directly on the Jenkins system.
```

Comment:
I would like to suggest to have some reasonable limit for the line length It is hard to read e.g. via github UI.